src/raftstore/store/worker/region.rs

```diff
@@ -140,7 +149,12 @@ impl SnapContext {
         timer.observe_duration();
     }
 
-    fn apply_snap(&self, region_id: u64, abort: Arc<AtomicUsize>) -> Result<()> {
+    fn apply_snap(
```

Comment:
I prefer to return the applied region info instead of passing the `pending_delete_ranges` in.